Archive Auction and Crowdloans (#6261)
w3f/polkadot-wiki: docs/build/build-parachains.md

These could be in the form of:

> - System level chains (permanent chains): via [agile coretime](../learn/learn-agile-coretime.md)

Reviewer: Agile coretime and related purchases do not apply to system chains

```suggestion
- System level chains (permanent chains)
```

### Parachain

To include your parachain into the Polkadot network, you will need to reserve some time on a relay

> chain core.

Reviewer: ```suggestion
To include your parachain into the Polkadot network, you will need to reserve a core on the relay
chain.
```

> Time on a core is acquired through [coretime](../learn/learn-agile-coretime.md) purchased with DOT

Reviewer: ```suggestion
[Coretime](../learn/learn-agile-coretime.md) can be purchased with DOT
```

Parachain slot leases are acuqired through open auctions, the mechanics of which can be found on the
[parachain auction](../learn/learn-auction.md) page. With
[Agile Coretime,](../learn/learn-agile-coretime.md) these parachain lease auctions will be
deprecated and parachains can [purchase coretime](../learn/learn-guides-coretime-marketplaces.md) to
produce blocks
[continuously or on-demand](../learn/learn-parachains.md#parachains-vs-on-demand-parachains).
Time on a core is acquired through [coretime](../learn/learn-agile-coretime.md) purchased with DOT
to produce blocks
[continuously or on-demand](../learn/learn-parachains.md#parachains-vs-on-demand-parachains) while
benefiting from Polkadot's security. See
[these guides](../learn/learn-guides-coretime-marketplaces.md) to learn how to purchase coretime.

## Resources

- [Parachain tutorials](https://docs.substrate.io/reference/how-to-guides/parachains/)

> - [System (Common Good) Parachains](https://polkadot.network/blog/common-good-parachains-an-introduction-to-governance-allocated-parachain-slots/)

Reviewer: This link can stay
